4. Making Decisions/java/Exercice 14/src/com/bravedroid/businessLogic/TaxCalculator.java

```diff
@@ -0,0 +1,14 @@
+package com.bravedroid.businessLogic;
+
+public class TaxCalculator {
+    private TaxCalculatorVM vm = new TaxCalculatorVM();
```

Comment:
visibility scope! : could be a variable you don't need it to be a field.

Reply:
👍 👍